api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -406,14 +408,25 @@
      * @throws ContextNotActiveException if there is no active context object for the given scope
      * @throws IllegalArgumentException if there is more than one active context object for the given scope
      */
-    public Context getContext(Class<? extends Annotation> scopeType);
+    Context getContext(Class<? extends Annotation> scopeType);
+
+    /**
+     * Creates a new {@link ManageableContext} that represents the given built-in scope.  This ManageableContext is not
+     * yet active, it must be activated when ready to use.  Upon activation, you may use it to retrieve beans from
+     * the context.
+     *
+     * @param scopeType the  {@linkplain javax.enterprise.context scope}
+     * @return the {@linkplain ManageableContext context object}
+     * @throws IllegalArgumentException if the given scopeType is not a valid built in scope.
+     */
+    ManageableContext createManageableContext(Class<? extends Annotation> scopeType);
```

Comment:
If that's still your opinion, then its not clear at all what this is trying to do and may be the root of why you and I are so far separated on this subject.  This method specifically creates a new `ManageableContext`, it's not bound to anything per se.  Repeated calls may end up activating the same context over and over again.  To put it another way...

```
ManageableContext c1 = beanManager.createManageableContext(RequestScoped.class);
ManageableContext c2 = beanManager.createManageableContext(RequestScoped.class);
assert !c1.equals(c2);
```